app/controllers/resumes_controller.rb

```diff
@@ -0,0 +1,38 @@
+class ResumesController < ApplicationController
+  def new
+    @resume = Resume.new
+    handler ||= RepositoryHandler.new(current_client, current_user)
+    if handler.populate_repositories
+      @repositories = Repository.all
+    else
+      flash[:errors] = "Couldn't hear back from github"
+      redirect_to profile_path
+    end
+  end
+
+  def create
+    resume = current_user.resumes.new(resume_params)
```

Comment:
This will explode when the current_user is nil

lets check for a user before we ask about resumes